routes/index.js

```diff
@@ -1,18 +1,18 @@
 const express = require('express')
 const app = express()
 
-app.use('/healthcheck', require('./healthCheck.js'))
+app.use('/auctions', require('./auctions.js'))
 app.use('/auth', require('./auth.js'))
-app.use('/users', require('./users.js'))
-app.use('/members', require('./members.js'))
-app.use('/tasks', require('./tasks'))
+app.use('/badges', require('./badges.js'))
 app.use('/challenges', require('./challenges.js'))
-app.use('/pullrequests', require('./pullrequests.js'))
 app.use('/contributions', require('./contributions'))
-app.use('/badges', require('./badges.js'))
-app.use('/trade', require('./trading.js'))
+app.use('/healthcheck', require('./healthCheck.js'))
+app.use('/members', require('./members.js'))
+app.use('/pullrequests', require('./pullrequests.js'))
 app.use('/stocks', require('./stocks.js'))
-app.use('/auctions', require('./auctions.js'))
+app.use('/tasks', require('./tasks'))
```

Comment:
`contributions` and `tasks` are not having `.js` extensions, can you please add?

Reply:
Sure I will do it